src/bioregistry/data/bioregistry.json

```diff
@@ -104533,6 +104460,79 @@
       "uri_format": "http://modelseed.org/biochem/reactions/$1"
     }
   },
+  "seed.subsystems": {
+    "biocontext": {
+      "prefix": "SEED.SUBSYSTEMS"
+    },
+    "contributor_extras": [
+      {
+        "email": "aireland@lbl.gov",
+        "github": "ialarmedalien",
+        "name": "AJ Ireland",
+        "orcid": "0000-0003-1982-9065"
+      },
+      {
+        "email": "seaver@anl.gov",
+        "github": "samseaver",
+        "name": "Sam Seaver",
+        "orcid": "0000-0002-7674-5194"
+      }
+    ],
+    "go": {
+      "description": "Project to annotate the first 1000 sequenced genomes, develop detailed metabolic reconstructions, and construct the corresponding stoichiometric matrices",
+      "homepage": "http://www.theseed.org",
+      "name": "The SEED;",
+      "prefix": "SEED.SUBSYSTEMS",
```

Comment:
@nalikapalayoor we shouldn't make any changes to anything inside the keys for the mapped prefixes, so e.g., the line above should stay the same as above.